README.md

```diff
@@ -1,24 +1,62 @@
-Camera server (PointGrey) and arduino based behaviour setup. 
+A `cmake` based pipeline to run behavioural experiments.
```

Comment:
Updated instructions in `README.md` file on how to run the pipeline. Please review and suggest changes. Any new user should be able to get started after reading this file.

Reply:
@anzalks Have a look at this README.md file if instructions make sense. If not, please do comment.